tests/backends/test_dbmodel.py

```diff
@@ -859,3 +859,83 @@ def test_inconsistent_params_db_model():
         ex.value.args[0]
         == "Cannot set devices_per_node>1 if CPU is specified under devices"
     )
+
+
+@pytest.mark.skipif(not should_run_tf, reason="Test needs TF to run")
+def test_db_model_ensemble_duplicate(fileutils, test_dir, wlmutils, mlutils):
+    """Test DBModels on remote DB, with an ensemble"""
+
+    # Set experiment name
+    exp_name = "test-db-model-ensemble-duplicate"
+
+    # Retrieve parameters from testing environment
+    test_launcher = wlmutils.get_test_launcher()
+    test_interface = wlmutils.get_test_interface()
+    test_port = wlmutils.get_test_port()
+    test_device = mlutils.get_test_device()
+    test_num_gpus = 1  # TF backend fails on multiple GPUs
+
+    test_script = fileutils.get_test_conf_path("run_tf_dbmodel_smartredis.py")
+
+    # Create the SmartSim Experiment
+    exp = Experiment(exp_name, exp_path=test_dir, launcher=test_launcher)
+
+    # Create RunSettings
+    run_settings = exp.create_run_settings(exe=sys.executable, exe_args=test_script)
+    run_settings.set_nodes(1)
+    run_settings.set_tasks(1)
+
+    # Create ensemble
+    smartsim_ensemble = exp.create_ensemble(
+        "smartsim_model", run_settings=run_settings, replicas=2
```

Comment:
very pedantic name change request
```suggestion
        "smartsim_ensemble", run_settings=run_settings, replicas=2
```